src/Html/HtmlCssSelectors.fs

```diff
@@ -0,0 +1,375 @@
+﻿namespace FSharp.Data
+
+open System
+open FSharp.Data
+open System.Runtime.CompilerServices
+
+module HtmlCssSelectors =
```

Comment:
Is this a part of the public interface? If so, it would be good to document it using `///`.

But as far as I can see, this is only internals that we don't need to expose. Could you mark it as `module internal HtmlCssSelectors = ...` ?